assets/stylesheets/_modules.scss

```diff
@@ -0,0 +1 @@
+@import 'partials/header';
```

Comment:
We should not have a `partials` folder. Too generic since most everything is a partial. Let's put it directly in `components`.